src/main/react-front-end/src/views/Login/CreateAccountComponent.js

```diff
@@ -127,10 +127,21 @@ export default class CreateAccountComponent extends Component {
     let self = this;
     let code = this.state.code;
     let state = self.state;
-
-    if (password !== confirmPassword) {
-      state.passwordError = "Password Doesn't Match";
-      self.setState({ state });
+    if(!(/[a-z]/.test(password))){
+			eventEmitter.emit("errorOccured", "Password must have atleast one lower character");
+		}
+		else if(!(/[A-Z]/.test(password))){
+			eventEmitter.emit("errorOccured", "Password must have atleast one upper character");
+		}
+		else if(!(/[0-9]/.test(password))){
+			eventEmitter.emit("errorOgccured", "Password must have atleast one digit");
```

Comment:
Spelling error.

Reply:
corrected spelling error and moved the function to constant.js